client/app/scripts/liveblog-edit/views/settings.html

```diff
@@ -49,7 +49,7 @@ <h2 translate>General</h2>
                                 <div class="form-input">
                                     <select
                                       ng-model="settings.blogPreferences.theme"
-                                      ng-options="theme as theme.name for theme in settings.availableThemes track by theme"
+                                      ng-options="theme.name as theme.name + ' ' for theme in settings.availableThemes"
```

Comment:
Because in the way described by you, the whole theme object is sent to the model. You have to send only the theme name. You are using "label for value in array track by..". I am doing "select as label for value in array". I added the trailing space because for some reason when the select expression evaluates the same as the label expression the "selected" attribute doesn't work well on the option